.github/workflows/auto-pr-submodules.yml

```diff
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
+name: Update Submodules and Submit a PR
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push  only for the actions-test-branch branches
```

Comment:
It seems that this workflow was copied from a template and this comment was not updated.